examples/experimental/dagster-airlift/README.md

```diff
@@ -1,26 +1,174 @@
+## REST API Availability
 
-Dagster is the “source of truth” for the state of migration as you can tell what assets and tasks have been fully migrated and which ones have not.
+Airlift depends on the the availability of Airflow’s REST API. Airflow’s REST API was made stable in its 2.0 release (Dec 2020) and was introduced experimentally in 1.10 in August 2018. Currently Airflow requires the availability of the REST API.
 
-This will greatly accelerate the migration because you can migrate tasks in parallel, in any order, with high trust, and do so aggressively because rolling back is easy, all within an orderly, observed, tool-assisted process.
+- **OSS:** Stable as of 2.00
+- **MWAA**
+  - Note: only available in Airflow 2.4.3 or later on MWAA.
+  - Doesn’t appear to be a well-defined reason other than pure conservatism according to [ChatGPT](https://chatgpt.com/c/220cd63e-2111-4b32-bd83-87d95e2390d7).
+- **Cloud Composer:** No limitations as far as we know.
+- **Astronomer:** No limitations as far as we know.
 
-## Trying out airlift
+## Peering
 
-See the peering process in action by trying out the example:
+The first step is to peer the Dagster Deployment and the Airflow instance.
 
-- [Airflow-Dagster Peering with DBT](examples/peering-with-dbt)
+To do this you need to install airlift in and make it available in your Dagster deployment.
+
+```bash
+uv pip install dagster-airlift[core]
+```
+
+At that point you create a `Definitions` object using `build_defs_from_airflow_instance`.
+
+```python
+from dagster_airlift.core import AirflowInstance, build_defs_from_airflow_instance
+
+defs = build_defs_from_airflow_instance(
+    airflow_instance=AirflowInstance(
+        auth_backend=BasicAuthBackend(
+            webserver_url="http://localhost:8080",
+            username="admin",
+            password="admin",
+        ),
+        name="airflow_instance_one",
+    )
+)
+```
+
+Note: When a code server is loaded for the first time it will query the target Airflow REST API. Subsequent process loads (e.g. a run worker loading) will used a cached response. If you want Dagster to pick up new DAGs, you will need restart the code server.
+
+An MWAA auth backend is available at `dagster_airlift.mwaa.auth.MwaaSessionAuthBackend`
+
+This function creates:
+
+- An asset representing each DAG. This asset is “materialized” whenever a DAG run completes.
+- A sensor that polls the Airflow instance for operational information. This is what keeps the DAG execution history up to date. It must be on to get timely information.
+
+### Peering to multiple instances
+
+Airlift supports peering to multiple Airflow instances, as you can invoke `create_airflow_instance_defs` multiple times and combine them with `Definitions.merge`
+
+```python
+from dagster_airlift.core import AirflowInstance, build_defs_from_airflow_instance
+
+defs = build_defs_from_airflow_instance(
```

Comment:
This isn't how definitions.merge works IIRC. It's actually `Definitions.merge(d1, d2)` (fully static).

I've been burned by this in the past